solr/solr-ref-guide/src/searching-nested-documents.adoc

```diff
@@ -0,0 +1,202 @@
+= Searching Nested Child Documents
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+This section exposes potential techniques which can be used for searching deeply nested documents,
```

Comment:
With a split content (index & search), I think it's quite important to link to each other.  So here at the start we need to link back to the indexing side.  On the index side page, somehow link to the search side -- perhaps at the start of that page too.

Reply:
There is a link to the indexing page in the following line.
Would that be sufficient?